server/src/main/scala/za/co/absa/atum/server/api/database/DoobieImplicits.scala

```diff
@@ -39,6 +39,36 @@ object DoobieImplicits {
 
   }
 
+  object HStore {
```

Comment:
There is support for hstore present in Doobie. Check this doc out. It's for the version the fa-db's doobie module depends on.
https://github.com/tpolecat/doobie/blob/v1.0.0-RC2/modules/docs/src/main/mdoc/docs/15-Extensions-PostgreSQL.md

Reply:
Hmm, okay, I didn't know, I'll play with it, thanks!

Reply:
done, thanks a lot